src/main/java/org/broadinstitute/hellbender/engine/ReadWalker.java

```diff
@@ -21,7 +22,7 @@
  * ReadWalker authors must implement the apply() method to process each read, and may optionally implement
  * onTraversalStart() and/or onTraversalSuccess(). See the PrintReadsWithReference walker for an example.
  */
-public abstract class ReadWalker extends GATKTool {
+public abstract class ReadWalker extends GATKTool implements CommandLinePluginProvider {
```

Comment:
You're right. Its going to wind up there anyway when the PR to move up read filters plugin goes in, so I'll move it there now.